tests/py/unit/test_tools_build.py

```diff
@@ -0,0 +1,41 @@
+from mat3ra.made.tools.build.interface import InterfaceDataHolder, patch_interface_with_mean_abs_strain
```

Comment:
This file should test the content of build.py - `create_interface`. The logic related to the interface data holder should be tested in a separate file